### PR TITLE
Add support for Anaconda

### PR DESCRIPTION
**Pull request by grsakea**
#### Pull Request (PR) description
This PR adds support for the anaconda distribution of python (https://www.anaconda.com/)

### CODE REVIEW

File: manifests/init.pp [hunk 1]
```diff
@@ -85,6 +85,8 @@
   Hash $python_dotfiles                           = { },
   Boolean $use_epel                               = $python::params::use_epel,
   $rhscl_use_public_repository                    = $python::params::rhscl_use_public_repository,
+  String $anaconda_installer_url                  = $python::params::anaconda_installer_url,
+  String $anaconda_install_path                   = $python::params::anaconda_install_path,
```
Comment (ekohl) [inline manifests/init.pp]: Could this be a `Stdlib::Absolutepath`?

File: manifests/init.pp [hunk 2]
```diff
@@ -85,6 +85,8 @@
   Hash $python_dotfiles                           = { },
   Boolean $use_epel                               = $python::params::use_epel,
   $rhscl_use_public_repository                    = $python::params::rhscl_use_public_repository,
+  String $anaconda_installer_url                  = $python::params::anaconda_installer_url,
```
Comment (ekohl) [inline manifests/init.pp]: Could this be a `Stdlib::Httpurl`?

File: manifests/virtualenv.pp [hunk 3]
```diff
@@ -96,7 +96,8 @@
   $extra_pip_args   = '',
   $virtualenv       = undef
 ) {
-  include ::python
```
Comment (ekohl) [inline manifests/virtualenv.pp]: Any reason this isn't needed anymore?

**Comment by othalla** [comment]
Dear maintainers, any update about this enhancement?

**Comment by bastelfreak** [comment]
Thanks for the PR @grsakea !

**Pull request merged**